### ISSUE TITLE
Installation progress bar resets to 0 on "Installing" step

### ISSUE THREAD

**Issue by mcclurgm** [elementary/appcenter#1501]
## What Happened

The progress bar jumps back to the start when the "installing" step starts. So the behavior looks like it fills halfway when downloading, then resets to 0, then fills halfway again when installing. I forgot to test this in the app details view when I still had updates, but I know it happens in the main list view.

## Expected Behavior

The installing step should continue the progress bar from half to full.

## Steps to Reproduce

1. Go to the Installed tab and install an update using one of the buttons in the list view.
2. Watch the progress bar that appears in that row.

## Platform Information

Odin daily, updated 3/5/21

**Comment by davidmhewitt** [comment]
Yeah I've seen this. I'm pretty sure it only affects packages installed via PackageKit though. Flatpak installations aren't affected by this.

**Comment by danirabbit** [comment]
Closing since we no longer support PackageKit 🎉
